src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -952,6 +954,7 @@ public static SyntaxTokenList GetModifiers(this SyntaxNode? member)
             {
                 case MemberDeclarationSyntax memberDecl: return memberDecl.WithModifiers(modifiers);
                 case AccessorDeclarationSyntax accessor: return accessor.WithModifiers(modifiers);
+                case LocalFunctionStatementSyntax localFunction: return localFunction.WithModifiers(modifiers);
```

Comment:
why is LocalDeclaration not handled here?

Reply:
@CyrusNajmabadi I wasn't able to write a test that hits this. Should I handle it without test coverage? or are you aware of any test case that will be affected?

Reply:
@CyrusNajmabadi Please confirm whether I should handle it here or not before merging.

Reply:
I think it's fine to handle this here